programs/launcher/main.cpp

```diff
@@ -24,6 +24,7 @@
 #include <sys/types.h>
 #include <netinet/in.h>
 #include <net/if.h>
+#include <eos/native_contract/native_contract_chain_initializer.hpp>
```

Comment:
Why was this added? I don't see it being used.

Reply:
I was planning on pulling producer names from the genesis block rather than just using "inita" etc. but I ran out of time.